.github/workflows/build-and-lint.yml

```diff
@@ -0,0 +1,44 @@
+name: Build and lint
+
+on:
+  pull_request:
+
+jobs:
+  sdk:
+    name: lint sdk
+    runs-on: ubuntu-20.04
+    env:
+      TEST_PATH: /tmp/test-results
+    steps:
+      - name: Checkout sdk
+        uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 14
+          cache: 'yarn'
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+      - name: Lint sdk
+        run: |
+          yarn build --reporter-options output=$TEST_PATH/sdk.xml
+          yarn lint --format junit -o $TEST_PATH/sdk-lint.xml
+      - name: Upload Coverage to Codecov
```

Comment:
I'm not sure if this is working properly, I see an warning when looking here: https://github.com/OffchainLabs/arbitrum-sdk/runs/8245403463?check_suite_focus=true

`Warning: Codecov: Failed to properly upload: The cwd: ./packages does not exist!`

Reply:
I removed the `work-directory` and it looks similar to that on circleci now.

Both said "No coverage report found. / No coverage files located" though. Should we remove this step?

Reply:
Hmm, yeh, looks like this hasnt been working for a while - if ever. Lets address it in another PR.